ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -34,24 +47,103 @@ export class ImportCommand extends IronfishCommand {
     },
   ]
 
+  async importLedger(): Promise<RpcAccountImport> {
+    const PATH = "m/44'/1338'/0"
+
+    const transport = await TransportNodeHid.create(3000, 3000)
+
+    const app = new IronfishApp(transport)
+
+    const appInfo = await app.appInfo()
+    if (appInfo.appName !== 'Ironfish') {
+      this.error('Please open the Iron Fish app on your ledger device')
```

Comment:
Why do we need to error out here? Can't we just wait for them to open the app and then continue?